### ISSUE TITLE
latest release 0.1.16 does not support GCS?

### ISSUE THREAD

**Issue by carolfly86** [wal-g/wal-g#208]
Hi, we were using pre-release version 0.2.3 to backup to GCS and tried to upgrade to the latest release 0.1.16. The 0.1.16 version throws error
```
FATAL: Did not set the following environment variables:
WALE_S3_PREFIX
```

**Comment by carolfly86** [comment]
seems like GCS support only available after 0.2.1. May I ask when will a stable version containing GCS support be released?

**Comment by x4m** [comment]
Hi!
v 0.2 will be released on 3th of April on no critical bugs found.
I understand that this is quite a big amount of time to wait. But we have to be precautious about releases.
